### PR TITLE
test_py_scripts - simplify get_py_script; add samples_path, get_data_dir

### PR DESCRIPTION
**Pull request by idanmiara**
## What does this PR do?

test_py_scripts - simplify get_py_script; add samples_path, get_data_dir and use them in tests instead of hardcode paths
This will allow adding/removing/changing Python script/modules paths easily.

This PR also cleanup some redundant spaces and empty lines from the effected files.

I'm not sure if FrankW's folder structure is relevant for anything. if not I'll remove it.

### CODE REVIEW

File: autotest/pymod/test_py_scripts.py [hunk 1]
```diff
@@ -37,20 +37,29 @@
...
+    return f'../{dir}/data/'
+
+
+def get_py_script(script_name):
+    # how to get to {root_dir}/gdal from ...
+    base_test_paths = [
+        # current layout (git) : {root_dir}/autotest/X
+        os.path.join(os.getcwd(), '..', '..', 'gdal'),
+        # old layout (FrankW's) : {root_dir}/gdal/autotest/X
+        os.path.join(os.getcwd(), '..', '..'),
```
Comment (rouault) [inline autotest/pymod/test_py_scripts.py:56]: I don't think that's relevant anymore indeed

Comment (idanmiara) [reply autotest/pymod/test_py_scripts.py]: OK. removed.

**Comment by idanmiara** [comment]
rebased on master

**Comment by idanmiara** [comment]
Thanks!

**Pull request merged**